yt_dlp/extractor/slideslive.py

```diff
@@ -139,6 +277,51 @@ def _real_extract(self, url):
         assert service_name in ('url', 'yoda', 'vimeo', 'youtube')
         service_id = player_info['service_id']
 
+        slides, slides_xml = None, None
+        slide_url_template = self._JPG_SLIDE_TMPL
+        chapters, thumbnails = [], []
+        if url_or_none(player_info.get('thumbnail')):
+            thumbnails.append({'url': player_info['thumbnail']})
+
+        if player_info.get('slides_json_url'):
+            slides = traverse_obj(self._download_json(
+                player_info['slides_json_url'], video_id, fatal=False,
+                note='Downloading slides JSON', errnote=False), 'slides', expected_type=list)
+        if slides:
+            if re.match(r'.+(/v[45]/).+', player_info['slides_json_url']):
+                slide_url_template = self._PNG_SLIDE_TMPL
+            for i in range(len(slides)):
+                slide_name = traverse_obj(slides, (i, 'image', 'name'))
+                slide_id = str(i + 1)
+                if slide_name:
+                    thumbnails.append({
+                        'id': slide_id,
+                        'url': slide_url_template % (video_id, slide_name),
+                    })
+                chapters.append({
+                    'title': slide_id,
+                    'start_time': int_or_none(traverse_obj(slides, (i, 'time')), scale=1000)
+                })
+
+        if not slides and player_info.get('slides_xml_url'):
```

Comment:
```suggestion
        elif player_info.get('slides_xml_url'):
```

Reply:
the request for the JSON URL can return 403 despite the URL existing in the API response. XML needs to be a fallback if JSON request fails

Reply:
Yes. The else is for above `if slides`

Reply:
nope, just me misremembering my own code. my bad. will apply